ci/config_solr.sh

```diff
@@ -0,0 +1,31 @@
+# Config variables local to the script
+version=${SOLR_VERSION:-6.6.3}
```

Comment:
does it matter that we have 6.6.5 in travis?

Reply:
I don't know, I just copied the script from another repo. That looks  a fallback value; @bwhicks should the default version be updated or does it matter?

Reply:
@rlskoeser Fall back value; Probably worth just changing it to use `SOLR_VERSION` as canonical.